README.md

```diff
@@ -69,7 +69,7 @@ Supported file formats are:
 
 *Or*
 
-`$ git clone https://kubevirt.io/containerized-data-importer.git`
+`$ git clone https://github.com/kubevirt/kubevirt.git` ('kubevirt.io' prefix)
```

Comment:
It'd be good to target the clone to the appropriate import path
e.g.
`$ git clone https://github.com/kubevirt/kubevirt.git $GOPATH/src/kubevirt.io/containerized-data-importer`

Reply:
Shouldn't it be: https://github.com/kubevirt/containerized-data-importer.git

Reply:
right. fixed.